packages/api/src/account/validators.test.ts

```diff
@@ -125,3 +129,138 @@ describe("validateUpdateTagsRequest", () => {
     }
   });
 });
+
+describe.skip("validateLeaveArchiveParams", () => {
+  test("should find no errors in valid parameters", () => {
+    let error = null;
+    try {
+      validateLeaveArchiveParams({
+        archiveId: "123",
+      });
+    } catch (err) {
+      error = err;
+    } finally {
+      expect(error).toBeNull();
+    }
+  });
+
+  test("should error if archiveId is missing", () => {
+    let error = null;
+    try {
+      validateLeaveArchiveParams({});
+    } catch (err) {
+      error = err;
+    } finally {
+      expect(error).not.toBeNull();
+    }
+  });
+
+  test("should error if archiveId is wrong type", () => {
+    let error = null;
+    try {
+      validateLeaveArchiveParams({
+        archiveId: 123,
+      });
+    } catch (err) {
+      error = err;
+    } finally {
+      expect(error).not.toBeNull();
+    }
+  });
+
+  test("should error if archiveId string contains non-numeric characters", () => {
```

Comment:
The name of this is a little confusing to me since the test value is all numeric.